zdg/mouse_motion_event.py

```diff
@@ -0,0 +1,26 @@
+import pygame
+from zdg.event import Event
+from zdg.settings import BLOCK_SIZE
+
+
+class MouseMotionEvent(Event):
+    def process_event(self):
+        super().process_event()
+
+        if not self.dragging_piece():
+            return
+
+        self.update_clicked_piece_location(self.event)
+        self.update_game_screen()
+        self.highlight_selected_piece()
+
+    def highlight_selected_piece(self):
+        screen = self.grid.screen
+        color = 'blue'
+        block_size = 0.6 * BLOCK_SIZE / 2
+        pos = (self.event.pos[0], self.event.pos[1])
```

Comment:
Just so I understand this a bit better, how is this different then `pos = self.event.pos`  or if it's a list: `pos = *self.event.pos`?

Reply:
You're right, pos = self.event.pos is better to use